cdap-docs/cdap-apps/source/hydrator/creating.rst

```diff
@@ -193,7 +193,275 @@ property replaces the ``schedule`` property of an ETL Batch Application.
 
 In the example code above, we will use a *ProjectionTransform* (a type of Transform) to drop certain
 columns in the incoming data. A *StreamSink* in the final step needs a data field property
-that it will use as the content for the data to be written. 
+that it will use as the content for the data to be written.
+
+Non-linear Executions in Pipelines
+----------------------------------
+
+ETL Applications support directed acyclic graphs in pipelines, which allows for the
+non-linear execution of pipeline stages.
+
+In this example, a pipeline reads from the stream ``purchaseStats``. It replicates the stream events
```

Comment:
'replicate' has special meaning in many technologies. Think its better just to say 'writes'